Intersect.Client/Interface/Game/DescriptionWindows/ItemDescriptionWindow.cs

```diff
@@ -0,0 +1,440 @@
+                var extraInfo = equipSlot;
+                if (mItem.EquipmentSlot == Options.WeaponIndex && mItem.TwoHanded)
+                {
+                    extraInfo = $"{Strings.ItemDescription.TwoHand} {equipSlot}";
+                }
+                header.SetSubtitle($"{typeDesc} - {extraInfo}", Color.White);
+            }
+            else
+            {
+                header.SetSubtitle(typeDesc, Color.White);
+            }
+
+            // Set up the item rarity label.
+            Strings.ItemDescription.Rarity.TryGetValue(mItem.Rarity, out var rarityDesc);
+            header.SetDescription(rarityDesc, rarityColor ?? Color.White);
+
+            header.SizeToChildren(true, false);
+        }
+
+        protected void SetupItemLimits()
+        {
+            // Gather up what limitations apply to this item.
+            var limits = new List<string>();
+            if (!mItem.CanBank)
+            {
+                limits.Add(Strings.ItemDescription.Banked);
+            }
+            if (!mItem.CanGuildBank)
+            {
+                limits.Add(Strings.ItemDescription.GuildBanked);
+            }
+            if (!mItem.CanBag)
+            {
+                limits.Add(Strings.ItemDescription.Bagged);
+            }
+            if (!mItem.CanTrade)
+            {
+                limits.Add(Strings.ItemDescription.Traded);
+            }
+            if (!mItem.CanDrop)
+            {
+                limits.Add(Strings.ItemDescription.Dropped);
+            }
+            if (!mItem.CanSell)
+            {
+                limits.Add(Strings.ItemDescription.Sold);
+            }
+
+            // Do we have any limitations? If so, generate a display for it.
+            if (limits.Count > 0)
+            {
+                // Add a divider.
+                AddDivider();
+
+                // Add the actual description.
+                var description = AddDescription();
+
+                // Commbine our lovely limitations to a single line and display them.
+                description.AddText(Strings.ItemDescription.ItemLimits.ToString(string.Join(", ", limits)), Color.White);
+            }
+        }
+
+        protected void SetupDescription()
+        {
+            // Add a divider.
+            AddDivider();
+
+            // Add the actual description.
+            var description = AddDescription();
+            description.AddText(Strings.ItemDescription.Description.ToString(mItem.Description), Color.White);
+        }
+
+        protected void SetupEquipmentInfo()
+        {
+            // Add a divider.
+            AddDivider();
+
+            // Add a row component.
+            var rows = AddRowContainer();
+
+            // Is this a weapon?
+            if (mItem.EquipmentSlot == Options.WeaponIndex)
+            {
+                // Base Damage:
+                rows.AddKeyValueRow(Strings.ItemDescription.BaseDamage, mItem.Damage.ToString());
```

Comment:
Is there a need to show damage even when it's 0?
It could be
![image](https://user-images.githubusercontent.com/63019821/144748248-639e25d8-87bc-4d08-a1e5-0ce84fa95277.png)

Reply:
I would argue that you always want to show the base damage and scaling values so people can do their own math.

Reply:
Really understood, thinking that way